build/Glyssen.proj

```diff
@@ -211,12 +211,18 @@
 	depending on whether your project's output MUST be signed or it's only nice if it is. For example, a continuous build
 	might use SignIfPossible, while a Publication build must use SignInstaller.-->
 	<Target Name="SignInstaller" DependsOnTargets="VersionNumbers; Installer">
-		<Exec Command='sign "$(RootDir)\output\installer\$(InstallerFileName)"'></Exec>
-		<Message Text="Signed installer at $(RootDir)\output\installer\$(InstallerFileName)" Importance="high"/>
+		<Exec Command='insignia -ib "$(InstallerPathWithVersion)" -o engine.exe'></Exec>
```

Comment:
I'm not familiar with insignia - I guess it gets installed as part of WIX?

Reply:
Yes, I think so. In the HearThis build log, this command results in the output: Windows Installer XML Toolset Inscriber version 3.11.2.4516...